modules/manage/pages/cluster-maintenance/compaction-tuning.adoc

```diff
@@ -0,0 +1,48 @@
+= Optimize Compaction Settings
+:description: Redpanda's approach to compaction and options for configuring it
+
+Configure compaction for your cluster to optimize storage utilization.
+
+== Repanda Compaction Overview
+
+Compaction is an optional mechanism intended to reduce the storage needs of Redpanda topics. Compaction is enabled through configuration of a cluster or topic's cleanup policy. When compaction is enabled as part the cleanup policy, a background process executes on a pre-set interval to perform compaction operations. When triggered for a partition the process purges older versions of messages for a given key and only retains the most chronologically recent message in that partition. This is done by analyzing closed segments in the partition, copying the most recent messages for each key into a new segment,then deleting the source segments.
+
+image::shared:compaction-example.png[Example of topic compaction]
+
+This diagram provides an illustration of a compacted topic. Imagine a remote sensor network that uses image recognition to track appearances of red pandas in an area and devices send messages to a topic when they detect one. You might enable compaction to reduce topic storage while still maintaining a record in the topic of the last time each device saw a red panda, perhaps to see if they stop frequenting a given area. The left side of the diagram shows all messages sent across the topic. The right side illustrates the results of compaction; older messages for certain keys are deleted from the message log.
+
+There are two important considerations for compaction:
+
+*  When using tiered storage, compaction functions at the local level before segments are uploaded to storage. Once a segment is uploaded to storage it is not retrieved for further compaction operations. A key may therefore appear in multiple segments between tiered and local storage. 
```

Comment:
> A key may therefore appear in multiple segments between tiered and local storage. 

This is true, but it is also true in all cases--compaction doesn't make guarantees about removing all duplicates. So if this sentence is intended to highlight a difference between the TS and non-TS scenarios I might rethink it.

EDIT: i see below you mention that there is no guarantee.